BUFR_TableD_en_09.csv

```diff
@@ -216,6 +216,35 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
 09,Vertical sounding sequences (conventional data),309026,(Single wavelength elastic backscatter lidar sequence),,033002,Quality information,,(19) The Quality information descriptor 0 33 002 relates to all parameters provided in the Backscatter data section and is given for each respective vertical bin of the profile measurement.,Operational
 09,Vertical sounding sequences (conventional data),309026,(Single wavelength elastic backscatter lidar sequence),,010071,Vertical resolution,,,Operational
 09,Vertical sounding sequences (conventional data),309026,(Single wavelength elastic backscatter lidar sequence),,027079,Horizontal width of sampled volume,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,301132,Common header sequence,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,008043,Atmospheric chemical or physical constituent type,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,025061,Software Version Number,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,302004,General cloud information,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,302005,Cloud layer,,,Operational
+09,Vertical sounding sequences (conventional data),309027,(Multi wavelength ground-based lidar sequence),,008092,Measurement uncertainty expression,,(18) The Measurement uncertainty expression and significance descriptors 0 08 092 and 0 08 093 relate to all parameter uncertainties provided in the backscatter data section.,Operational
```

Comment:
@amilan17 I suggest here that (18) be replaced by (see Note 18)

Reply:
done